mltrace/entities/component_run.py

```diff
@@ -26,6 +26,7 @@ def __init__(
         inputs: typing.List[IOPointer] = [],
         outputs: typing.List[IOPointer] = [],
         git_hash: str = None,
+        git_tags: typing.List[str] = [],
```

Comment:
It might be better to keep the `git_tags` type in the client data model here consistent with the ORM data model, which stores `git_tags` as a String currently in this PR. One option to avoid hassle is to keep this client data model with a list type and change the column in the ORM data model to be of `PickleType`. See the `stale` attribute for an example.

Reply:
updated to PickleType. Thanks for the example for reference ~

Reply:
awesome